pom.xml

```diff
@@ -74,7 +74,10 @@
         <skin.groupId>org.apache.maven.skins</skin.groupId>
         <skin.artifactId>maven-fluido-skin</skin.artifactId>
         <skin.version>2.0.0</skin.version>
-        <netbeans.version>RELEASE220</netbeans.version>
+        <!-- version of Apache NetBeans for harness build -->
+        <netbeans.version>RELEASE230</netbeans.version>
+        <!-- version of Apache NetBeans artefacts that pass jdk8 it -->
+        <netbeans.version.it>RELEASE220</netbeans.version.it>
     </properties>
```

Comment:
would this work?
```xml
<netbeans.version.it>${netbeans.version}</netbeans.version.it>
```
and
```xml
    <profiles>
        <profile>
            <id>downgrade-release-for-jdk8-it</id>
            <activation>
                <jdk>(,11]</jdk>
            </activation>
            <properties>
                <netbeans.version.it>RELEASE220</netbeans.version.it>
            </properties>
        </profile>
    </profiles>
```

Reply:
You're not going to convince me that shouldn't be `RELEASE126` .. but whatever works.

Reply:
as indicated on the slack conversation I don't really care about the number, I agree whatever works is fine.

To do this properly though the range would have to be adjusted to `12]` so that it covers the JDK 11 matrix axis too.

Reply:
yes, that is exactly what i said on slack. this needs 2 profiles. thats why I wouldn't bother. If JDK 8 and 11 tests work on RELEASE220 - lets keep testing on it.

Reply:
Yes, you don't think it matters, I do.  But not enough to car to reiterate the same conversation here! :smile:  I'm neither approving nor blocking the proposed change.